src/app/apps/vacunacion/components/gestion-turnos/monitoreo-inscriptos.ts

```diff
@@ -126,6 +131,11 @@ export class MonitoreoInscriptosComponent implements OnInit {
         } else {
             this.gruposPoblacionales = [];
         }
+        this.conceptosTurneablesService.getAll().subscribe(data => {
```

Comment:
Buscar el concepto por id y evitar traer todos los conceptos de la api.
```suggestion
        this.conceptosTurneablesService.search({id: this.idVacunacionCovid}).subscribe(data => {
```
En algunos casos como https://github.com/andes/app/blob/master/src/app/apps/rup/mapa-camas/constantes-internacion.ts, el concepto directamente está como constante del lado de la app. Evaluar mejor opción.